packages/fuchsia_ctl/test/emulator_test.dart

```diff
@@ -0,0 +1,142 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:file/memory.dart';
+import 'package:mockito/mockito.dart';
+import 'package:process/process.dart';
+import 'package:test/test.dart';
+
+import 'package:fuchsia_ctl/fuchsia_ctl.dart';
+
+import 'fakes.dart';
+
+void main() {
+  const String aemuPath = '/emulator';
+  const String fuchsiaImagePath = '/fuchsia.blk';
+  const String fuchsiaSdkPath = '/fuchsia-sdk';
+  const String qemuKernelPath = '/qemu-kernel.kernel';
+  const String zbiPath = '/zircon-a.zbi';
+
+  Emulator emulator;
+  MockProcessManager mockProcessManager;
+  MemoryFileSystem fs;
+
+  setUp(() {
+    mockProcessManager = MockProcessManager();
+    fs = MemoryFileSystem(style: FileSystemStyle.posix);
+    fs.file(aemuPath).createSync();
+    fs.file(fuchsiaImagePath)
+      ..createSync()
+      ..writeAsString('fuchsia image content');
+    fs.file(fuchsiaSdkPath).createSync();
+    fs.file(qemuKernelPath).createSync();
+    fs.file(zbiPath).createSync();
+    emulator = Emulator(
+      aemuPath: aemuPath,
+      fs: fs,
+      fuchsiaImagePath: fuchsiaImagePath,
+      fuchsiaSdkPath: fuchsiaSdkPath,
+      processManager: mockProcessManager,
+      qemuKernelPath: qemuKernelPath,
+      zbiPath: zbiPath,
+    );
+  });
```

Comment:
Can we add the tests to a group? is easier to run all when using an IDE.

Reply:
Added